conceptual/Npgsql/release-notes/8.0.md

```diff
@@ -64,6 +64,27 @@ Version 8.0 contains many other smaller features and bug fixes, [see the 8.0.0 m
 
 ## Breaking changes
 
+### Synchronous command batching can cause threadpool starvation
+
+To allow Npgsql to write command batches containing large data payloads (generally parameter data) synchronously, without running into PostgreSQL limitations, it runs any commands after the first asynchronously. This introduces sync over async and can cause threadpool starvation at higher loads. In Npgsql versions prior to 8.0 a workaround was in place that would mitigate the effects of this sync over async, allowing most higher loads to run succesfully. This workaround was removed to make room for the far reaching NativeAOT refactor.
```

Comment:
Should probably add a few examples how threadpool starvation affects users, like delayed query execution time and even timeouts.